packages/react-hooks/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## [Unreleased] -->
 
+## [1.5.0] - 2020-01-22
+
+- Fixed some typos in `useMountedRef` hook https://github.com/Shopify/quilt/pull/1243
```

Comment:
If we're just making `README.md` changes, we don't need to publish a new version or note it in the changelog really. We [ignore readme](https://github.com/Shopify/quilt/blob/master/lerna.json#L10) changes in our lerna config so these changes won't get picked up anyways.

Reply:
ah ok will remove